libsolidity/ast/Types.cpp

```diff
@@ -1145,9 +1145,14 @@ TypePointer RationalNumberType::binaryOperatorResult(Token _operator, TypePointe
 
 				if (exp >= 0)
 					value = rational(numerator, denominator);
-				else
+				else {
```

Comment:
`{` to the next line.

Reply:
Too long that I haven't written solidity code :-D.